src/pykeen/models/unimodal/node_piece.py

```diff
@@ -103,7 +128,20 @@ def __init__(
                 "representations inverse relation representations are required.",
             )
         # normalize embedding specification
-        embedding_specification = embedding_specification or EmbeddingSpecification(shape=(embedding_dim,))
+        anchor_specification = embedding_specification or EmbeddingSpecification(
+            shape=(embedding_dim,),
+            initializer=entity_initializer,
+            normalizer=entity_normalizer,
+            constrainer=entity_constrainer,
+            regularizer=entity_regularizer,
+        )
+        relation_specification = EmbeddingSpecification(
```

Comment:
you're sure this shouldn't be `embedding_specification or EmbeddingSpecification(`?

Reply:
Please double check this, then it looks good to me!

Reply:
We initialize `anchor_specification` in this manner - so by default the sent specification is considered to be anchor specification. The reason for that is that anchor and relation specifications in most cases will use different initialization and normalization strategies, we can't use it at the same time for both